src/Sarif.Multitool/appsettings.local.json

```diff
@@ -0,0 +1,2 @@
+{
+}
```

Comment:
If you want to override the appsettings, use this file. Anything in here will override the values in appsettings.json. I also added this file to .gitignore. So you shouldn't have to worry about checking in secrets.

Reply:
Does it make sense to also have setting in the SarifWorkItemContext?  This is where we allow our configuration file to setup and override other keys, though this may be a problem the context is not initialized when we first need the logger.

Reply:
I wouldn't have expected these settings to be settable from the SarifWorkItemContext. But per the standup conversation, I'd prefer to defer this decision until we have a better understanding of how and where we handle configuration.

Reply:
I wouldn't have expected these settings to be settable from the SarifWorkItemContext. But per the standup conversation, I'd prefer to defer this decision until we have a better understanding of how and where we handle configuration.

---
In reply to: [396154828](https://github.com/microsoft/sarif-sdk/pull/1820#discussion_r396154828) [](ancestors = 396154828)